.nvmrc

```diff
@@ -1 +1 @@
-v12
+12.4
```

Comment:
This doesn't match the Dockerfile: https://github.com/artsy/metaphysics/blob/7cc291d16db51c7cc03c12b35840eb5f0dbe3010/Dockerfile#L1